network/gossip/libp2p/middleware.go

```diff
@@ -223,6 +238,24 @@ func (m *Middleware) Stop() {
 	m.wg.Wait()
 }
 
+// TEMP fix till PR that changes overlay.Topology to return a flow.IdentityList instead of a map is merged to master
```

Comment:
this will be removed after PR https://github.com/onflow/flow-go/pull/48 is merged to master